examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,125 @@
+        Logger.log(Logger.Level.INFO, "app", "Set response is " + setResponse.get());
+
+        CompletableFuture<String> getResponse = client.get("foo");
+        Logger.log(Logger.Level.INFO, "app", "Get response is " + getResponse.get());
+
+        // Send PING to the primary node
+        CompletableFuture<String> pong = client.ping();
+        Logger.log(Logger.Level.INFO, "app", "Ping response is " + pong.get());
+    }
+
+    /** Executes the application logic with exception handling. */
+    private static void execAppLogic() {
+
+        while (true) {
+            GlideClient client = null;
+            try {
+                client = createClient();
+                appLogic(client);
+
+            } catch (CancellationException e) {
+                Logger.log(Logger.Level.ERROR, "glide", "Request cancelled: " + e.getMessage());
+                throw e;
+            } catch (InterruptedException e) {
+                Logger.log(Logger.Level.ERROR, "glide", "Client creation interrupted: " + e.getMessage());
+                Thread.currentThread().interrupt(); // Restore interrupt status
+                throw new CancellationException("Interrupted while creating client");
+            } catch (ExecutionException e) {
+                // All Glide errors will be handled as ExecutionException
+                if (e.getCause() instanceof ClosingException) {
+                    // If the error message contains "NOAUTH", raise the exception
+                    // because it indicates a critical authentication issue.
+                    if (e.getMessage().contains("NOAUTH")) {
+                        Logger.log(
+                            Logger.Level.ERROR, "glide", "Authentication error encountered: " + e.getMessage());
+                    } else {
+                        Logger.log(
+                            Logger.Level.WARN,
+                            "glide",
+                            "Client has closed and needs to be re-created: " + e.getMessage());
+                    }
+                } else if (e.getCause() instanceof ConnectionException) {
+                    // The client wasn't able to reestablish the connection within the given retries
+                    Logger.log(Logger.Level.ERROR, "glide", "Connection error encountered: " + e.getMessage());
+                } else if (e.getCause() instanceof TimeoutException) {
+                    // A request timed out. You may choose to retry the execution based on your application's
+                    // logic
+                    Logger.log(Logger.Level.ERROR, "glide", "Timeout encountered: " + e.getMessage());
+                } else if (e.getCause() instanceof ExecAbortException) {
+                    Logger.log(Logger.Level.ERROR, "glide", "ExecAbort error encountered: " + e.getMessage());
+                } else {
+                    Logger.log(Logger.Level.ERROR, "glide", "Execution error during client creation: " + e.getCause());
```

Comment:
Same as above, is the execution exception handled here specific to client creation or can it also happen inside appLogic? If it can happen inside appLogic we should change the message to "Execution error encountered: "

Reply:
If it could happen in appLogic can we change to "Execution error encountered: "?

Reply:
Miss this one. I will update it now.